pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -85,6 +85,9 @@ type Manager interface {
 	// and is consulted to achieve NUMA aware resource alignment per Pod
 	// among this and other resource controllers.
 	GetPodTopologyHints(pod *v1.Pod) map[string][]topologymanager.TopologyHint
+
+	// GetAllocatableCPUs returns the assignable (not allocated) CPUs
+	GetAllocatableCPUs() []int64
```

Comment:
I think this should return a `cpuset.CPUSet` just like the call to `GetCPUs()` above.
Also, can you move this call just below `GetCPUs()` to group them together better.

Reply:
ack

Reply:
Do we want, in a future, lower priority PR, change the return type of `GetCPUs` so it returns `cpuset.CPUSet`, to make the code consistent and improve the flexibility?

Reply:
If it's not too much effort, I think it would be good to do it now.

Reply:
Fair enough. I'll see what I can do (in a separate commit)